.circleci/config.yml

```diff
@@ -109,10 +113,67 @@ jobs:
           path: ~/project
       - attach_workspace:
           at: /tmp/workspace
-      - run: ../scripts/pack_deb.sh -v 0.1 -f /tmp/workspace/administration/build -d "Administration backend for the Ehrenamtskarte app" -n "eak-administration"
+      - run: ~/project/scripts/pack_deb.sh -v "0.$CIRCLE_BUILD_NUM" -f /tmp/workspace/administration/build -d "Administration backend for the Ehrenamtskarte app" -n "eak-administration"
       - run: |
           mkdir -p /tmp/artifacts/debs
-          cp *.deb /tmp/artifacts/debs
+          mv *.deb /tmp/artifacts/debs
+      - store_artifacts:
+          path: /tmp/artifacts
+      - persist_to_workspace:
+          root: /tmp/artifacts
+          paths:
+            - debs/*.deb
+
+  styles-pack:
+    docker:
+      - image: cimg/base:2020.01
+    working_directory: ~/project/mapbox/styles
+    steps:
+      - checkout:
+          path: ~/project
+      - run:
+          name: "Download glyphs"
+          command: |
+            wget -O /tmp/glyphs.zip https://github.com/orangemug/font-glyphs/archive/gh-pages.zip
+            unzip /tmp/glyphs.zip
+            mv font-glyphs* font-glyphs
+            rm -f /tmp/glyphs.zip
+      - run:
+          name: "Build .deb"
+          command: ~/project/scripts/pack_deb.sh -v "0.$CIRCLE_BUILD_NUM" -m . -d "Mapbox styles.json with dependencies for the Ehrenamtskarte app" -n "eak-styles"
+      - run:
+          name: "Move .deb to artifacts folder"
+          command: |
+            mkdir -p /tmp/artifacts/debs
+            mv *.deb /tmp/artifacts/debs
+      - store_artifacts:
+          path: /tmp/artifacts
+      - persist_to_workspace:
+          root: /tmp/artifacts
+          paths:
+            - debs/*.deb
+
+  martin-pack:
+    docker:
+      - image: cimg/base:2020.01
+    working_directory: ~/project/mapbox/martin
+    steps:
+      - checkout:
+          path: ~/project
+      - run:
+          name: "Download martin"
+          command: |
+            downloadUrl=$(curl -sH "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/urbica/martin/releases/latest" | grep -o "http.*martin-Linux.*.tar.gz")
+            echo "Downloding and extracting $downloadUrl …"
+            wget -q -O- "$downloadUrl" | tar -xz
```

Comment:
It would be very cool if we would build martin from source here. Else there is not really an advantage to the current salt deployment.

Reply:
Sure, I can do that. What is the problem with the pre-compiled Martin?

Reply:
Its just that this is not an improvement but more complexity in the deployment.
The martin from source has severy security fixes. Also it is more "trustworthy" than that binary.

Reply:
But I think this does not really have priority.

Reply:
Done